### ISSUE TITLE
Item is object,  checking object equality with '===' will not get expected result.

### ISSUE THREAD

**Issue by snannan268** [maslianok/react-responsive-tabs#77]
https://github.com/maslianok/react-responsive-tabs/blob/c2e30b7957dfafd94459b09ffb473f1e5ee1f0e0/src/index.js#L68

As the item is object,  so use '===' approach to check the equality will get an incorrect result.

More detail could reference: http://adripofjavascript.com/blog/drips/object-equality-in-javascript.html

Or for here, we could jsut check the title changes or not, since the tab width just related to the title.
Such as:
```
items.every((item, i) => item.title !== prevProps.items[i].title)
```

**Comment by maslianok** [comment]
It doesn't matter what type of equality you use.
The condition returns `false` for different instances and `true` for the same instance.

But I tend to agree with your suggested change: the `title` prop is a kind of required prop and I can't imagine someone uses the module without it. So it's better to check this prop instead of object instances.

Thank you, @snannan268 for your efforts 👍

**Comment by maslianok** [comment]
Published in v4.2.0
